tests/spam_methods_test.py

```diff
@@ -23,6 +23,29 @@
     CorrectionMethod,
 )
 from qermit.taskgraph import CircuitShots  # type: ignore
+from qermit.spam import gen_UnCorrelated_SPAM_MitRes
+from qermit import CircuitShots
+from pytket import Circuit
+from pytket.extensions.quantinuum import QuantinuumBackend
+from .test_backends import MockQuantinuumBackend
+import pytest
+
+
+@pytest.mark.xfail(
+    reason=("Presently pytket removes unused qubits. There is also a missmatch between node and qubit names with QuantinuumBackend")
+)
+def test_mock_quantinuum_all_qubits():
```

Comment:
`def test_mock_quantinuum_all_qubits() -> None`

Reply:
Is there something special about this test that means it requires type annotation? I don't think we have done this elsewhere.

Reply:
Oh no, if we haven't done it elsewhere don't worry about it. I've been `mypy` pinged for it before on `TKET` so was trying to get ahead of that warning here.